public/sass/_grafana.scss

```diff
@@ -1,5 +1,23 @@
+// This is specified in runtime Emotion GlobalStyles but we need them for the preloader styles
+html {
+  font-size: $font-size-base;
+  height: 100%;
+}
+
+// This is specified in runtime Emotion GlobalStyles but we need them for the preloader styles
+body {
+  font-family: $font-family-sans-serif;
+  font-size: $font-size-base;
+  line-height: $line-height-base;
+  color: $text-color;
+  background-color: $body-bg;
+  height: 100%;
+  width: 100%;
+  margin: 0;
+  position: absolute;
+}
+
```

Comment:
we need to keep these in SCSS for now since they're needed for the preloader (before emotion has even loaded). haven't figured out a plan for these yet 🤔

Reply:
Write them directly into the html file? 🤔

Reply:
or just plain old boring .css file 😌

Reply:
yeah boring .css file might be the best - will try in a follow up PR.

writing them directly into the html file has a disadvantage - they then have a higher precedence when applying the styles. when in the SCSS, any of the styles from emotion take precedence, so it works nicely because these styles get applied, then emotion overwrites them with the proper styles. putting them in a style tag in the html file means that we'd need to add `!important` to the corresponding styles in emotion